### ISSUE TITLE
"ss(...).dirtyForms is not a function" error on every admin page

### ISSUE THREAD

**Issue by mmenozzi** [Sylius/Sylius#10669]
**Sylius version affected**: 1.6.0

**Description**  
In every page of the admin there is the following javascript error:

```
Uncaught TypeError: ss(...).dirtyForms is not a function
    at HTMLDocument.<anonymous> (app.js:1)
    at e (app.js:1)
    at t (app.js:1)
```

**Steps to reproduce**  
Login into admin and open any page (even the dashboard).

**Comment by igormukhingmailcom** [comment]
Same for me... Have anyone a solution?

**Comment by maximehuran** [comment]
Do you have this error in a Sylius upgrade or after a fresh installation of Sylius 1.6 ?

**Comment by igormukhingmailcom** [comment]
> Do you have this error in a Sylius upgrade or after a fresh installation of Sylius 1.6 ?

After upgrade

**Comment by maximehuran** [comment]
I have the same thing after upgrade too.

On the Sylius 1.6 demo this error does not exists : https://demo.sylius.org/admin/

**Comment by maximehuran** [comment]
Done : https://github.com/Sylius/Sylius/compare/master...maximehuran:patch-1

**Comment by pamil** [comment]
Fixed by #10760.
